specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/livySpark.json

```diff
@@ -705,14 +705,17 @@
       "type": "object",
       "properties": {
         "from": {
+          "description": "The start index to fetch sessions.",
           "format": "int32",
           "type": "integer"
         },
         "total": {
+          "description": "Number of sessions to fetch.",
           "format": "int32",
           "type": "integer"
         },
         "sessions": {
+          "description": "Batch list.",
```

Comment:
>Batch list [](start = 26, length = 10)

The Spark Batch job list.

Reply:
Thanks for your reminder. I will scan all description again to enhance the completeness.